Shivani/HTML/firstassignment4.html

```diff
@@ -1,9 +1,7 @@
 <!DOCTYPE html>
-
 <html>
 <head>
 <title>assign4</title>
-
 </head>
 <body bgcolor="lightblue">>
 <h1 style="color:tomato;"> Heading1 </h1>
```

Comment:
in assignment 4 you have tell about yourself not just tags adding from internet ok